schema.sql

```diff
@@ -1,5 +1,10 @@
 /* Database schema to keep the structure of entire database. */
 
 CREATE TABLE animals (
-    name varchar(100)
+  id INT,
```

Comment:
- [ ] [OPTIONAL] You can make this id column not null and autoincremented like so 👇🏾 

```sql
    id int NOT NULL IDENTITY(1,1)
```
- [ ] [OPTIONAL] You can also mark the id column as a primary key like so 👇🏾 

```sql
    PRIMARY KEY (id)
```
So your code will be like so 👇🏾 

```sql
CREATE TABLE animals (
     id int NOT NULL IDENTITY(1,1),
    name varchar(255),
    date_of_birth date,
    escape_attempts int,
    neutered boolean,
    weight_kg decimal, -- numeric data type is also accepted
    );
   PRIMARY KEY (id)
```

Reply:
Thank you so much for the suggestions.